unit_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# cd to the script directory:
+cd "${0%/*}" || { echo "Couldn't cd to ${0%/*}!"; exit 1; }
+
+export UNIT_TESTS_DIR=./build/out/bin
+
+${UNIT_TESTS_DIR}/bwl_dummy_unit_tests
+
```

Comment:
Sorry, I'm not sure I can understand your comment. If you mean `unit_tests.sh` in root and inside tools/docker, this was the existing approach. I just did copy&paste

Reply:
Yes, this is what I meant `unit_tests.sh` and `tools/docker/unit_tests.sh` (I forgot to mention it explicitly, my bad).
Even if it _is_ the approach (can you give an example?) - I don't think that doing wrong should be copy/pasted. However, I am not sure it is wrong, I just ask for reconsideration.
My main concern is for new developers: when you are new to a project it takes time to understand the structure and the logic. Finding two `unit_tests.sh` scripts under the source tree _might_ look like leftovers, unclean source tree, something that was here in the past and was copied instead of moved. Also one should _guess_ that one unit-test runs in docker and the second is for running locally. It is _clear_ once you are familiar. But I feel that we have many of these that are not **crystal** clear, and I try to minimize them.
_Please take into account that the above paragraph might be wrong_

Reply:
Your paragraph is right and I fully agree with you (it takes a while to discover how it works because it's not clear). 
However I think it's better to stick to the plan until a new better design is agreed. In any case such re-design of native vs docker scripts naming convention is out of the scope of this PR.